### ISSUE TITLE
Getting started: set aria-checked on the top level buttons

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#117336]
Refs: https://github.com/microsoft/vscode/issues/117313

The top level button elements should have [`aria-checked`](https://w3c.github.io/aria/#aria-checked) set on them since they are big checkboxes at the end that track your progress. That way the screen reader will read `checked` or  `unchecked` when the user navigates to one of them and then they can know if they finished this task or not.

![Screenshot 2021-02-23 at 09 59 52](https://user-images.githubusercontent.com/1926584/108821440-e2d7e300-75bd-11eb-8b19-0a48f5f1fb22.png)

**Comment by isidorn** [comment]
Opening this since I still see sometimes that aria-checked = undefeind. It should always be `true` or `false`. So maybe just `!!` it.

**Comment by JacksonKearl** [comment]
Good catch!

**Comment by isidorn** [comment]
This works nice now, adding verified label
